packages/adminrouter/extra/src/test-harness/tests/test_master.py

```diff
@@ -365,3 +365,20 @@ def test_if_xaccel_header_is_passed_to_client_by_ar(
             valid_user_header,
             '/system/v1/logs/foo/bar',
             assert_headers=accel_buff_header)
+
+    def test_if_dcos_flavor_is_open(
```

Comment:
Admin Router test harness is not suited for this test - can we have here a dc/os integration test, please ?

Admin Router tests use mock data to test various endpoints, the content of this data is not as important as the fact that the data was correctly served(headers, response content, upstream request, failure conditions, etc...). Integration test would on the other hand test that `gen` indeed returns the correct "flavour" for a given dc/os variant.

Let me know if you need some help writing such a test.

Reply:
Can we use: dcos_distribution

Reply:
In the same vein, maybe it should be more than just saying "open" vs. "enterprise". And specifically "Mesosphere DC/OS" and "Mesosphere Enterprise DC/OS"

Reply:
ACK. `distribution` is it then.

@vespian thanks. I'll move it.